src/main/ts/rules/PreferFun.ts

```diff
@@ -0,0 +1,76 @@
+import { Rule } from 'eslint';
+import { ArrowFunctionExpression, Expression, FunctionExpression, Literal, Statement } from 'estree';
+import * as path from 'path';
+
+export const preferFun: Rule.RuleModule = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Prefer using the Katamari Fun helper functions, instead of inline functions.'
+    },
+    messages: {
+      preferNoop: 'Prefer using `Fun.noop` instead of `() => {}`',
```

Comment:
I wonder if these messages will be confusing in code like this (I don't know why someone would write this): 

```
function () { 
   return; 
};
```

Maybe...

```suggestion
      preferNoop: 'Prefer using `Fun.noop` instead of declaring a new no-op function',
```

I can't really think of an alternative

Reply:
The code is clearer to me tbh, though I get what you mean. I'd almost do "e.g. `() => {}`" but that's weird English. Not sure what's best here.

Reply:
Yeah, I went with the more common denominator here. I could technically use templates to attempt to determine whether it's `() => {}` or `function () { return; }`, but didn't see it as worthwhile.

Reply:
```
This function is a duplicate of `Fun.noop` — prefer using that instead
```

Thoughts?

Reply:
I like the direction this is going in...

Reply:
I wanted to keep it starting with `Prefer` as that's the common convention eslint uses.

Reply:
Actually reading again, the convention they use is:
```
Use {name} instead of {other}
```

So whatever we use, my condition is that it should stick to that.

Reply:
Maybe we could use something like this?
```
Use `Fun.noop` instead of creating a new no-op function
```

Reply:
Rephrased as discussed on slack.